### DISCUSSION TITLE
How to run SQL queries to the connected database?

### DISCUSSION THREAD

**Discussion by Sheharyar566** [jahlborn/jackcess#11] [Q&A] [answered]
I think the title explains it all. But I tried looking into the QueryImpl and the other derivative classes but couldn't figure out how to run sql queries.

**Comment by jahlborn** [comment, marked as answer]
> I think the title explains it all. But I tried looking into the QueryImpl and the other derivative classes but couldn't figure out how to run sql queries.

https://jackcess.sourceforge.io/faq.html#query
